pandas/core/internals/managers.py

```diff
@@ -367,6 +368,7 @@ def apply(
         out = type(self).from_blocks(result_blocks, self.axes)
         return out
 
+    # TODO: Cleanup?
     # Alias so we can share code with ArrayManager
     apply_with_block = apply
```

Comment:
Should this alias be removed?

Reply:
it can go, as can all of inernals.base IIRC

Reply:
Makes sense. Because part of the implementation needs to be moved from internals.base, I'd like to take that on in a followup.